### DISCUSSION TITLE
키글챌 22회차 반성문

### DISCUSSION THREAD

**Discussion by mun-kyeong** [BDD-CLUB/challenge-every-week-posting#239] [6. 반성문]
알고보니 21회차때도 잊고 못적었더군요..!! ㅜㅜ 
늘 블로깅을 다짐하면서도 실행에 옮기는 일이 정말 어려운 것 같습니다..

이전엔 티스토리에서 글을 작성하다 이번에 velog로 블로그를 이전하게 되었습니다.
사실 git 블로그를 만들고 싶었는데 너무 바쁘다보니 자꾸만 미루게 되더군요... (환경세팅이 젤 싫어요)
그러다 현재 제게 중요한건 플랫폼이 아니라 블로깅 글쓰기 습관이라는 생각이 들었습니다. 
언젠가 시간이 된다면 git블로그를 다시 도전하겠지만 그 전까지는 velog에서 글쓰기 습관을 들이려고 합니다.

이번주부터는 꾸준히 써보겠습니다..!!

**Comment by shkisme** [comment]
글쓰기 습관 형성 쉽지 않더라구요...ㅎㅎ (아직도)
비글챌로 화이팅 해봅시다 !!!
